.woodpecker/deploy_content.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+  - name: ci
+    when:
```

Comment:
wondering if the `volumes` section is missing here or left out on purpose

Reply:
With woodpecker CI there is no general `volumes` section, only one for each step. In other repos we're using our own docker image, which includes a composer configuration and setup with caching. We're not (yet) using that here, so at the moment no volume is needed in the step